armsrc/util.c

```diff
@@ -305,3 +305,11 @@ bool data_available(void) {
     return usb_poll_validate_length();
 #endif
 }
+
+bool data_available_fast(void) {
```

Comment:
Can clarity be improved as to meaning of this new function?

Those not really familiar with `data_available()` might see this new function and think it's a 100% replacement, only faster!  If so, a comment mentioning this is all that is needed.

If it's not a 100% replacement, consider comments that clarify:
* what is meant by "fast"
* are there invariants that are assumed by this function (e.g., what conditions must be true for this to be safe to call?)
* when would callers want to call the non-fast version instaed of this one?

Reply:
I see. I've put the comments in the `armsrc/util.c`